### PR TITLE
Fix wording in integrating third party asset guide

### PR DESCRIPTION
**Pull request by IsaacMurtagh**
The wording around payment method, asset type, and payment option was inconsistent.

### CODE REVIEW

**Review by alexBinns — COMMENTED** [review]
Should it be payment option or payment method? It seems that payment option is used more in the docs.

File: src/content/guides/integrating-third-party-asset.mdx [hunk 1]
```diff
@@ -46,15 +46,15 @@ An example description is `Centrapay NZD`, `Bitcoin`, or `Ethereum`.
 
 ### Supported Currencies
 
-Payment Requests have a value determined by the currency a Merchant accepts. You should supply a finite list of three-letter [ISO currency codes](https://en.wikipedia.org/wiki/ISO_4217).
+[Payment Requests](https://docs.centrapay.com/api/payment-requests) have a value determined by the currency a Merchant accepts. You should supply a finite list of three-letter [ISO currency codes](https://en.wikipedia.org/wiki/ISO_4217) that is supported by your asset.
 
-### Example Payment Option
+### Example Defintiation
```
Comment (daallen303) [inline src/content/guides/integrating-third-party-asset.mdx:51]: I think this change may have been unintentional.

**Comment by alexBinns** [comment]
> 

I guess there are certain contexts where each of them make sense one would be in a more technical aspect and the other would be in a less technical aspect

File: src/content/guides/integrating-third-party-asset.mdx (hunk 1 above)
Comment (IsaacMurtagh) [reply src/content/guides/integrating-third-party-asset.mdx:51]: Yes, thanks :)

Comment (IsaacMurtagh) [reply src/content/guides/integrating-third-party-asset.mdx:51]: It was intentional. The H2 is called Defining a payment method, so thought it was redundant to add Payment Method to this header

**Review by laurencemcknight — APPROVED** [review]

**Pull request merged**